src/pkg/k8s.go

```diff
@@ -170,7 +170,7 @@ func (s *JobRunner) getPodObject(identifier string, labels map[string]string, jo
 			InitContainers: []corev1.Container{
 				{
 					Name:            "helper",
-					Image:           fmt.Sprintf("public.ecr.aws/opslevel/opslevel-runner:v%s", ImageTagVersion),
+					Image:           "public.ecr.aws/opslevel/opslevel-runner:v2024.1.3", // TODO: fmt.Sprintf("public.ecr.aws/opslevel/opslevel-runner:v%s", ImageTagVersion),
```

Comment:
this sucks but for now its acceptable to hard code it. We need to find a better solution long term for locadev.